### PR TITLE
3d test new

### PR DESCRIPTION
**Pull request by shyeavr**
Add concurrent test, fix one point per user as Alex ask , redesign the percent calculation, retype env functions, add payload flag to config

### CODE REVIEW

File: playground.py
Comment (shay-perpi) [inline playground.py:1]: Delete this file from git

Comment (shay-perpi) [inline playground.py:1]: Delete this file

File: tests/elevation_service/graph_check.py
Comment (shay-perpi) [inline tests/elevation_service/graph_check.py:1]: Delete this File

File: tests/elevation_service/graph_function.py
Comment (shay-perpi) [inline tests/elevation_service/graph_function.py:1]: Delete this File

File: tests/elevation_service/one_point_per_user.py [hunk 1]
```diff
@@ -1,131 +1,167 @@
...
+    initiate_counters_by_ranges,
+    retype_env,
+    write_rps_percent_results,
 )
 
 positions_list_path = ElevationConfig.positions_path
 
-if type(ElevationConfig.percent_ranges) == str:
-    percent_ranges = list(ElevationConfig.percent_ranges)
+if isinstance(ElevationConfig.percent_ranges, str):
```
Comment (shay-perpi) [inline tests/elevation_service/one_point_per_user.py:18]: put in config

File: tests/elevation_service/one_point_per_user.py [hunk 2]
```diff
@@ -1,131 +1,167 @@
...
-        #     write_rps_percent_results(
-        #         custom_path=reports_path, percente_value_by_range=percent_value_by_range
-        #     )
-        # else:
-        #     percent_value_by_range["total_requests"] = total_requests
+        # points = polygon_random_points(polygon=ElevationConfig.poly, num_points=1)
+        body = generate_points_request(points_amount=1, polygon=ElevationConfig.poly, exclude_fields=True,
+                                       product_type="MIXED")
+        # body = json.loads(next(self.request_bodies_cycle))
+        print(body)
```
Comment (shay-perpi) [inline tests/elevation_service/one_point_per_user.py:57]: Delete print

**Review by shay-perpi — COMMENTED** [review]
Delete

File: tests/elevation_service/one_point_per_user.py (hunk 2 above)
Comment (shyeavr) [reply tests/elevation_service/one_point_per_user.py:57]: This is the old one..

**Pull request closed without merging**